packages/frontend/editor-ui/src/router.ts

```diff
@@ -788,7 +788,8 @@ router.beforeEach(async (to: RouteLocationNormalized, from, next) => {
 		 */
 
 		await initializeCore();
-		await initializeAuthenticatedFeatures();
+		// Pass undefined for first param to use default
+		await initializeAuthenticatedFeatures(undefined, to.name as string);
```

Comment:
Passing the first param as undefined to use the default value. Is there a better way to do this?